frontend/public/components/factory/table.tsx

```diff
@@ -86,6 +87,7 @@ const sorts = {
   dataSize: (resource) => _.size(_.get(resource, 'data')) + _.size(_.get(resource, 'binaryData')),
   ingressValidHosts,
   serviceCatalogStatus,
+  instanceType: (obj): string => getMachineSetInstanceType(obj),
```

Comment:
Note that the new table from #8829 doesn't require you to add a filter here. It might be better for this use case, but this is OK for now.